src/main/java/com/baidu/hugegraph/api/graphs/GraphsAPI.java

```diff
@@ -54,19 +81,30 @@ protected String type() {
         return result.readList(this.type(), String.class);
     }
 
+    public void clear(String graph, String message) {
+        String path = String.join("/", this.path(), graph, "clear");
+        this.client.delete(path,
+                           ImmutableMap.of("confirm_message", message));
+    }
+
+    public void delete(String graph, String message) {
+        this.client.checkApiVersion("0.67", "dynamic graph delete");
+        this.client.delete(joinPath(this.path(), graph),
+                           ImmutableMap.of(CONFIRM_MESSAGE, message));
+    }
+
     public void mode(String graph, GraphMode mode) {
-        String path = String.join("/", this.path(), graph);
         // NOTE: Must provide id for PUT. If use "graph/mode", "/" will
         // be encoded to "%2F". So use "mode" here although inaccurate.
-        this.client.put(path, "mode", mode);
+        this.client.put(joinPath(this.path(), graph), MODE, mode);
```

Comment:
can add joinPath(path, graph, action) method: joinPath(this.path(), graph, MODE),

Reply:
PUT method must have `id`, MODE is `id`, should not treated as path